doc/source/dependency-management.rst

```diff
@@ -125,3 +135,143 @@ Here are some examples of runtime environments combining multiple options:
    :language: python
    :start-after: __runtime_env_conda_def_start__
    :end-before: __runtime_env_conda_def_end__
+
+.. _remote-uris:
+
+Remote URIs
+-----------
+
+The ``working_dir`` and ``py_modules`` arguments in the ``runtime_env`` dictionary can specify either local path(s) or remote URI(s).
+
+A local path must be a directory path. The directory's contents will be directly accessed as the ``working_dir`` or a ``py_module``.
+A remote URI must be a link directly to a zip file. **The zip file must contain only a single top-level directory.**
+The contents of this directory will be directly accessed as the ``working_dir`` or a ``py_module``.
+
+For example, suppose you want to use the contents in your local ``/some_path/example_dir`` directory as your ``working_dir``.
+If you want to specify this directory as a local path, your ``runtime_env`` dictionary should contain:
+
+.. code-block:: python
+
+    runtime_env = {..., "working_dir": "/some_path/example_dir", ...}
+
+Suppose instead you want to host its contents remotely and provide a remote URI.
+You would need to first compress the ``example_dir`` directory into a zip file.
+There should be no other files or directories at the top level of the zip file, other than ``example_dir``.
```

Comment:
Is it possible to give a quick example of how to do this, like an example command or something?